x/stake/keeper/delegation.go

```diff
@@ -36,7 +36,44 @@ func (k Keeper) GetAllDelegations(ctx sdk.Context) (delegations []types.Delegati
 	return delegations
 }
 
-// load all validators that a delegator is bonded to or retrieve a limited amount
+// Return all validators that a delegator is bonded to. If maxRetrieve is supplied, the respective amount will be returned
```

Comment:
Awesome! Can you update this comment and the rest to be punctuated at the end (it's a sentence)?